pytest_tank_test/tk_fixtures.py

```diff
@@ -0,0 +1,232 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2020 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+import pytest
+import os
+from tk_toolchain.authentication import get_toolkit_user
+from tk_toolchain.testing import create_unique_name
+
+
+@pytest.fixture(scope="session")
+def tk_test_shotgun():
+    """
+    Getting credentials from TK_TOOLCHAIN
+    """
+    sg = get_toolkit_user().create_sg_connection()
+
+    return sg
+
+
+@pytest.fixture(scope="session")
+def tk_test_current_user(tk_test_shotgun):
+    """
+    Get current user
+
+    :returns: The current user id and name
+    """
+    user = get_toolkit_user()
+    username = tk_test_shotgun.find_one(
+        "HumanUser", [["login", "is", str(user)]], ["name"]
+    )
+
+    return username
+
+
+@pytest.fixture(scope="session")
+def tk_test_create_project(tk_test_shotgun):
```

Comment:
I think `tk_test_project` would suffice here. This is not a function you call directly to create a project, you're asking for the `tk_test_project` fixture to be passed to your test when you write. What do you think?

```
def foo(tk_test_project):
    pass
```